src/Commands/sql/SanitizeContactFormFieldsCommands.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_contact_forms\Commands\sql;
+
+use Drush\Commands\DrushCommands;
+use Drush\Drupal\Commands\sql\SanitizePluginInterface;
+use Consolidation\AnnotatedCommand\CommandData;
+use Symfony\Component\Console\Input\InputInterface;
+use Drupal\Core\Database\Connection;
+use Drupal\Core\Extension\ModuleHandler;
+use Drupal\Core\Logger\LoggerChannelFactoryInterface;
+
+/**
+ * Sanitizes the contact forms related data.
+ */
+class SanitizeContactFormFieldsCommands extends DrushCommands implements SanitizePluginInterface {
+
+  /**
+   * The database service.
+   *
+   * @var \Drupal\Core\Database\Connection
+   */
+  protected $database;
+
+  /**
+   * The module handler service.
+   *
+   * @var \Drupal\Core\Extension\ModuleHandler
+   */
+  protected $moduleHandler;
+
+  /**
+   * The logger service.
+   *
+   * @var \Drupal\Core\Logger\LoggerChannelInterface
+   */
+  protected $logger;
+
+  /**
+   * SanitizeContactFormFieldsCommands constructor.
+   *
+   * @param \Drupal\Core\Database\Connection $database
+   *   The database service.
+   * @param \Drupal\Core\Extension\ModuleHandler $moduleHandler
+   *   The module handler service.
+   * @param \Drupal\Core\Logger\LoggerChannelFactoryInterface $loggerFactory
+   *   The logger service.
+   */
+  public function __construct(Connection $database, ModuleHandler $moduleHandler, LoggerChannelFactoryInterface $loggerFactory) {
+    $this->database = $database;
+    $this->moduleHandler = $moduleHandler;
+    $this->logger = $loggerFactory->get('oe_contact_forms');
+  }
+
+  /**
+   * Sanitize the contact data from the DB.
+   *
+   * @hook post-command sql-sanitize
+   *
+   * @inheritdoc
+   */
+  public function sanitize($result, CommandData $commandData) {
+    $this->database->update('contact_message')
+      ->expression('name', "CONCAT('User', id)")
+      ->expression('mail', "CONCAT('user+', id, '@example.com')")
+      ->expression('subject', "CONCAT('subject-by-', id)")
+      ->expression('message', "CONCAT('message-by-', id)")
+      ->expression('oe_country_residence', "CONCAT('residence-in-', id)")
+      ->expression('oe_telephone', "CONCAT('+000-', id)")
+      ->expression('oe_topic', "CONCAT('topic-', id)")
+      ->fields([
+        'ip_address' => '127.0.0.1',
+      ])
+      ->execute();
+
+    $this->logger->notice('Contact messages data has been sanitized.');
```

Comment:
I am sorry @georgebaev . Yesterday by mistake have removed more correct comment. It shouldn't be 'notice' but 'success':
```suggestion
    $this->logger->success('Contact messages data sanitized.');
```

Reply:
@sergepavle this is fixed.

Reply:
@sergepavle , drone fails, and if we look at https://github.com/php-fig/log/blob/master/Psr/Log/LoggerInterface.php there is not success method.
We could use info().

Reply:
The `info()` method does not display any message in the `$ drush sql-sanitize` command result log. I reverted back the logger message type to `notice`.

Reply:
I just suggested make messaging more consistent with other sanitization commands where used logger:
\Drush\Drupal\Commands\sql\SanitizeSessionsCommands::sanitize